### PR TITLE
Add unit test for Object3D and Camera

### PR DESCRIPTION
**Pull request by simonThiele**
As discussed last evening here is the PR with some unit tests for Object3D.

### CODE REVIEW

**Comment by arose** [comment]
see #7108

File: test/unit/core/Object3D.js [hunk 1]
```diff
@@ -0,0 +1,116 @@
...
+ */
+
+module( "Object3D" );
+
+test( "rotateX", function() {
+  var obj = new THREE.Object3D();
+
+  // get a reference object for comparing
+	var reference = obj.rotateOnAxis(new THREE.Vector3(1, 0, 0), 45).rotation;
+  obj.rotateX(45);
```
Comment (WestLangley) [inline test/unit/core/Object3D.js]: `rotateX()` calls `rotateOnAxis()`. You are not actually testing anything here.

This appears to be an issue throughout this PR.

Also, Euler rotations are not unique. I do not think it is wise to compare them. Perhaps comparing the quaternions is best. But even so, the negation of a unit quaternion represents the same rotation.

**Comment by WestLangley** [comment]
@simonThiele Thank you for working on the unit tests. It is important!

**Comment by mrdoob** [comment]
Do you mind going a "Indent spaces to tabs" in these files? We try to use tabs for intenting.

**Comment by simonThiele** [comment]
yep, using tabs now.

File: test/unit/core/Object3D.js [hunk 2]
```diff
@@ -0,0 +1,102 @@
...
+	obj.lookAt(new THREE.Vector3(0, -1, 1));
+
+	ok( obj.rotation.x * RadToDeg === 45 , "x is equal" );
+});
+
+test( "getWorldQuaternion", function() {
+	var obj = new THREE.Object3D();
+
+	obj.lookAt(new THREE.Vector3(0, -1, 1));
+	ok( obj.getWorldRotation().x * RadToDeg === 45 , "x is equal" );
```
Comment (WestLangley) [inline test/unit/core/Object3D.js]: `getWorldRotation()` or `getWorldQuaternion()`?

Comment (simonThiele) [inline test/unit/core/Object3D.js]: getWorldQuaternion! thx

**Comment by mrdoob** [comment]
Sweet! Thanks!

**Pull request merged**